src/Processing.js

```diff
@@ -4066,10 +4066,36 @@
     p.noLoop = function() {
       doLoop = false;
       loopStarted = false;
-      clearInterval(looping);
+      
+      looping = false;
       curSketch.onPause();
     };
 
+requestAnimationFrame= (function(){
+      return window.requestAnimationFrame ||
+             window.webkitRequestAnimationFrame ||
+             window.mozRequestAnimationFrame    ||
+             function( callback ){
+                var _h= function(){callback(Date.now());};
+                window.setTimeout(_h, curMsPerFrame);
+             };
+     })();
+
+    var _now, _then=0, _doDraw = function(t) {
```

Comment:
these `_now` and `_then` feel like they should be inside a closure for the doDraw function, which isn't a temporary value as far as I can see, so probably doesn't need the leading underscore:
```
var doDraw = (function() {
  var now =  ..., then = ...;
  return function(t) {
    ...
  };
}());
```